verifier/verifier.py

```diff
@@ -229,26 +223,29 @@ def retrieve_models(self, uri, model_dir):
         """
         Function to obtain pre-trained model for verification predictions
         """
-        with file_locker.FileLocker('model_op.lock'):
```

Comment:
Don't we need this file lock when gunicorn uses multiple processes?

Reply:
Fixed.